src/index.html

```diff
@@ -3,10 +3,380 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Catalog</title>
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page">
+    <header class="header">
+      <div class="header__container container">
+        <a href="#home" class="logo">
+          <img src="./images/logo.png" alt="MOYO logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list nav__list--hidden">
+          <li class="nav__item">
+            <a class="nav__item-link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item" data-qa="nav-hover">
+            <a class="nav__item-link " href="#Laptops&Computers">Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="#gadgets">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="#tablets">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="#photo">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="#video">Video</a>
+          </li>
+        </ul>
+        <div class="menu-btn1">
```

Comment:
You are now adding a style, but not formatting it in any way. Do not add what you do not use.